.gitmodules

```diff
@@ -17,3 +17,6 @@
 	path = deps/libsrtp
 	url = https://github.com/cisco/libsrtp.git
 	shallow = true
+[submodule "deps/mbedtls"]
+	path = deps/mbedtls
+	url = https://github.com/Mbed-TLS/mbedtls.git
```

Comment:
Is Mbed TLS added as a submodule just because Ubuntu only provides the version 2 ?

For the macOS, you use homebrew like for OpenSSL.

Reply:
Unfortunately I can't use Mbed TLS from homebrew or apt.

`MBEDTLS_SSL_DTLS_SRTP` isn't enabled by default. I opened a PR [here](https://github.com/Homebrew/homebrew-core/pull/125402) to enabled it for homebrew.

Reply:
@paullouisageneau I agree, but I think this is the 'least' worst?

* Users that don't enable Mbed TLS aren't paying a cost (add_subdirectory doesn't happen)
* Keeps CI+Development simple. Having a pinned version of Mbed TLS and making development easy is nice

In the future if Homebrew and Ubuntu package Mbed TLS with DTLS-SRTP enabled we could drop it then?

Reply:
There is still a cost for users because Mbed TLS will be cloned by default, for instance when you run `git clone --recursive` or `git submodule --init`.  Since the Homebrew package now has DTLS-SRTP enabled (thank you for this), I think for now you could remove Ubuntu which doesn't feature MbedTLS 3 anyway and drop the submodule.

Reply:
@paullouisageneau Done! Removed as submodule, and will install as system lib in github action.